BDArmory/FX/ShellCasing.cs

```diff
@@ -21,8 +22,7 @@ void OnEnable()
             startTime = Time.time;
             velocity = initialV;
             velocity += transform.rotation *
-                        new Vector3(Random.Range(-.1f, .1f), Random.Range(-.1f, .1f),
-                            Random.Range(6f, 8f));
+                        (new Vector3(Random.Range(-.1f, .1f), Random.Range(-.1f, .1f),  Random.Range(-.1f, .1f)) + configV);
```

Comment:
This changes the range of the `.z` component from 6—8 to 6.9—7.1.
For consistency with how it used to be and fewer calls to `Random` (which is expensive), something like
```csharp
Vector3 direction = Random.insideUnitSphere;
velocity += transform.rotation * new Vector3(0.1f * direction.x, 0.1f * direction.y, direction.z + configV);
```
would work.

Additionally, the `angularVelocity` line below would be better as
```csharp
angularVelocity = 100f * Random.insideUnitSphere;
```